runtime/Cpp/runtime/src/TokenStreamRewriter.h

```diff
@@ -5,6 +5,8 @@
 
 #pragma once
 
+#include "antlr4-common.h"
```

Comment:
Why's that here now?

Reply:
This is to make the header compilable by itself, rather than relying on include order. This is necessary for some tooling.